repos/spack_repo/builtin/packages/chapel/package.py

```diff
@@ -938,6 +951,15 @@ def test_chpldoc(self):
             # TODO: Need to update checkChplDoc to work in the spack testing environment
             pass
 
+    def test_mason(self):
+        """Run the mason test"""
+        if not self.spec.satisfies("+mason"):
+            print("Skipping mason test as mason variant is not set")
+            return
+        else:
+            # TODO: Need to create a mason test for the spack testing environment
+            pass
```

Comment:
I realize this procedure is templated off the (currently useless) `test_chpldoc`. That being said, I don't love the proliferation of dead/useless framework code. In particular, I believe as written both of these give the end user the misleading impression that something has been successfully tested for chpldoc/mason, when in fact nothing was actually tested. 

In chpldoc's case IIRC there was originally real test code there that was later disabled due to technical problems with the hopes of re-enabling it "soon", but since that has not happened we should probably consider removing that test. That's orthogonal to this PR, but please don't clone that code until/unless there's some real meaningful mason test code available that can run in Spack.